File: edu/HRO/TIRPRO01 Programmeren 1/Lua/o3.lua
-- opdracht 3; Priemgetallen
local i = 0 -- index of the prime number
local p = 1 -- number to be checked

while i < 200 do
	p = p + 1
	local isprime = true
	
	for j = 2, p-1 do
		if p%j == 0 then
			isprime = false
			break
		end
	end
	
	if isprime then
		i = i + 1
		print(i, p)
	end
end

File: edu/HRO/TIRPRO01 Programmeren 1/o3.java
// opdracht 3; Priemgetallen
public class o3
{
	public static void main(String[] args)
	{
		int i = 0; // index of the prime number
		int p = 1; // number to be checked
		
		while (i < 200)
		{
			p++;
			boolean isprime = true;
			
			for (int j=2; j<p; j++)
			{
				if (p%j == 0)
				{
					isprime = false;
					break;
				}
			}
			
			if (isprime)
			{
				i++;
				System.out.println(i+"\t"+p);
			}
		}
	}
}

